mod/quiz/accessrule/seb/classes/access_manager.php

```diff
@@ -179,6 +240,29 @@ private function get_this_page_url() : string {
         return $FULLME;
     }
 
+    /**
+     * Returns reason for access prevention as a text.
+     *
+     * @param string $identifier Reason string identifier.
+     * @return string
+     */
+    private function get_reason_text(string $identifier) : string {
+        if (in_array($identifier, ['not_seb', 'invalid_config_key', 'invalid_browser_key'])) {
+            return get_string($identifier, 'quizaccess_seb');
+        }
+
+        return get_string('unknown_reason', 'quizaccess_seb');
+    }
+
+    /**
+     * Get list of allowed browser exam keys.
+     *
+     * @return array Browser exam keys. Empty if not using browser exam keys.
+     */
+    public function get_valid_browser_exam_keys(): array {
```

Comment:
I can't see this method is used anywhere. Did you replace it with valid_browser_exam_key_exists?

Reply:
Left over during a few refactors. Removed.